### ISSUE TITLE
Request for free disk space

### ISSUE THREAD

**Issue by carcade** [johman10/flood-for-transmission#288]
Hello! It would be nice, if you can add some widget showing free space on disk.

**Comment by johman10** [comment]
As of right now the Transmission API allows for this, but I would want to show this in relation to the total space on disk. I created a PR for Transmission to add this information: https://github.com/transmission/transmission/pull/1682, it was merged, but a new version of Transmission was not released yet so I'm on hold for that.

I can keep this ticket open until it's been released and I wil implement this free/total space widget.

**Comment by carcade** [comment]
Great! Thanks for the fast feedback!

**Comment by carcade** [comment]
Hello! Is there any news on this feature?

**Comment by johman10** [comment]
I'm still waiting on a new release to be created for Transmission so that [the API that I added](https://github.com/transmission/transmission/pull/1682) becomes available.

I would recommend watching the [Transmission releases](https://github.com/transmission/transmission/releases), once a new release comes out I can properly look into this feature. Until then I don't want to release any feature based on the master branch of Transmission.

**Comment by carcade** [comment]
Sorry, I've seen that this transmission feature is "merged", but didn't mention that it's not in release yet. Thanks!

**Comment by Gnaagounet** [comment]
Hello, new release is out for Transmission, do you have any news about this feature ?

**Comment by johman10** [comment]
@Gnaagounet as you can see i started work on this here: https://github.com/johman10/flood-for-transmission/pull/519. But it's not done yet.

Can't make any promises currently regarding timeline or anything.

**Comment by Gnaagounet** [comment]
Thanks, not critical feature, it is in the backlog :)

**Comment by johman10** [comment]
The goal is to keep this ui working with Transmission 3, but this feature won't work until you upgrade.

There is probably some more work left to make it look good etc and then to make sure that everything works all around. But yeah, they are the major ones I believe.

I don't think checking every minute is of value, but of course this can be made configurable as well. Before I add more config I would like to merge @defkev's changes in the configuration setup though.

**Comment by warmach1ne** [comment]
I'm running the 4.1.0 dev version of transmission. I hope it comes soon. Thanks for the work @johman10!!
